app/components/status-badge.jsx

```diff
@@ -0,0 +1,66 @@
+// https://github.com/EnCiv/civil-pursuit/issues/47
+
+'use strict'
+import React from 'react';
+import { createUseStyles } from 'react-jss';
+import cx from 'classnames';
+
+function StatusBadge(props) {
+    const { style = {}, className = "", name = "", status = "", number = undefined, ...otherProps } = props;
+    const classes = useStyles();
+
+    const statusMapping = {
+        progress: "Unread",
+        complete: "Complete",
+        inactive: "Group, Inactive",
+        error: "Most Important"
+    }
+
+    return (
+        <span className={cx(classes.container, classes[status.toLowerCase()])}>
+            <span className={classes.status}>{statusMapping[status.toLowerCase()]}</span>
```

Comment:
the idea of having the "name" prop was that the name would be rendered here (rather than statusMapping[] , and the status prop would indicate how it should be rendered.  If we use only one prop "status" then we constrain how this component can be used elsewhere.